src/main/java/com/siit/hospital_manager/model/Patient.java

```diff
@@ -17,6 +17,8 @@ public class Patient extends User{
     private String name;
     private Integer age;
     private String phoneNumber;
+    private String medicalHistory;
```

Comment:
I'm pretty sure a medical history should contain more that 1 string. List<Strings> would be a bare minimum, but a List<MedicalHistory> objects with specific field (diagnostic, diagnosedDate, isCured, List<Medicine> prescription> would be much better